src/entities/profile/api/types.ts

```diff
@@ -1,3 +1,8 @@
+interface IUserFavorite {
+  id: number,
+  name: string
+}
+
 export type ProfileDto = {
```

Comment:
Use an interface instead of a type.eslint

https://habr.com/ru/sandbox/186102/ - types vs interfaces